### ISSUE TITLE
[Filter panel]: Web Components: Prod QA testing of the `View all` link for filter groups

### ISSUE THREAD

**Issue by RobertaJHahn** [carbon-design-system/carbon-for-ibm-dotcom#7407]
#### User Story
> As a `[user role below]`:
developer using the Carbon for IBM.com `Filter panel`

> I need to:
have a version of the component that has been tested for accessibility compliance as well as on multiple browsers and platforms 

> so that I can:
be confident that my ibm.com web site users will have a good experience

#### Additional information
- [Browser Stack link](https://ibm.ent.box.com/notes/578734426612)
- [Browser Standard](https://w3.ibm.com/standards/web/browser/) 
- Sanity test of tier 1 mobile browsers (desktop is covered in e2e tests)
- Sanity test of Accessibility (Voiceover)
- [Accessibility testing guidance](https://pages.github.ibm.com/IBMa/able/Test/verify/)
 - [Accessibility Checklist](https://www.ibm.com/able/guidelines/ci162/accessibility_checklist.html)
 - [Creating a QA bug](https://ibm.ent.box.com/notes/603242247385)
- **See the Epic (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7376) for the Design and Functional specs information**
 - Web Components Dev issue (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7405) 
- Once development is finished the updated code is available in the [**Web Components Canary Environment**](https://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/overview-getting-started--page) for testing.
- [**Web Components canary storybook**](https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components)
- [**React canary storybook**](https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/react)
- [**React wrapper storybook**](https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-react)

#### Acceptance criteria
- [ ] Accessibility testing is complete
- [ ] All manual testing is complete
- [ ] Defects are recorded

**Comment by jeffchew** [comment]
@kennylam @oliviaflory , don't think I'll have time for this one, going to unassign myself from this.

**Comment by proeung** [comment]
@ljcarot Our QA team went through each component within the WC package for the recent release, however, I don't believe they were assigned to this issue to verify that the "View all" link passed all of the requirements outlined in this issue.

Should we assume that any new feature requests that our Innovation team is working on should go through our QA queue as well? If so, we'll make sure to note this in the next upcoming release as I don't believe this was something we anticipated in the first round of testing that we did for `v.1.31.0`, but we'll be happy to help out any way that we can move forward.

cc: @QALosey @raximkoron
